proxy/src/http/websocket.rs

```diff
@@ -234,6 +235,8 @@ pub async fn task_main(
         info!("websocket server has shut down");
     }
 
+    let conn_pool: &'static Arc<GlobalConnPool> = Box::leak(Box::new(GlobalConnPool::new(config)));
```

Comment:
Not sure why we need double static here, (`&'static` and `Arc`)
```suggestion
    let conn_pool: Arc<GlobalConnPool> = GlobalConnPool::new(config);
```

Reply:
Without that extra static `async move` in `make_service_fn` does not compile. I haven't tried to properly understand why, so may be there is a better way.

Reply:
Ok, with extra clone in `make_service_fn` this thing works without static.